examples/src/app/ApplicationImpl.ts

```diff
@@ -169,6 +171,17 @@ export class ApplicationImpl implements Application {
         unit: 'm',
     });
 
+    readonly testProgress: Progress = new ProgressImpl({
+        id: 'ApplicationImpl.progress',
+        initialValue: 80,
+    });
+
+    readonly testProgress2: Progress = new ProgressImpl({
+        id: 'ApplicationImpl.progress',
```

Comment:
Shouldn't we use separate IDs for the two test progresses?

Reply:
Ok